config/load.go

```diff
@@ -7,12 +7,17 @@ import (
 	"strings"
 	"sync"
 
-	"github.com/BuxOrg/bux-server/dictionary"
+	"github.com/bitcoin-sv/spv-wallet/dictionary"
 	"github.com/mitchellh/mapstructure"
 	"github.com/rs/zerolog"
 	"github.com/spf13/viper"
 )
 
+const (
+	// EnvPrefix is the prefix for environment variables
+	EnvPrefix = "SPV"
```

Comment:
🤔 why actually we have separated const `ConfigEnvPrefix` in config.go 
Maybe we can have only one?

Beside that:
```suggestion
	EnvPrefix = "SPV_WALLET"
```